packages/paste-website/src/components/empty-state/Error404.tsx

```diff
@@ -25,20 +26,17 @@ const Error404 = ({pathname, componentList, utilitityList}: Error404Props): Reac
       }
       return <NotBuilt type="component" name={pageName} />;
     }
-
-    if (pathname.includes(SidebarCategoryRoutes.UTILITIES)) {
-      if (isInDevelopment) {
-        return <InDevelopment type="utility" name={pageName} />;
-      }
-      return <NotBuilt type="utility" name={pageName} />;
-    }
   }
 
-  /* 
+  /*
   // TODO populate graphql first
   if (pathname.includes(SidebarCategoryRoutes.PRIMITIVES) && primitiveList.includes(packageName)) {
       <NotBuilt type="primitive" name={name} />;
   }
+
+  if (pathname.includes(SidebarCategoryRoutes.LAYOUT) && layoutList.includes(packageName)) {
+      <NotBuilt type="layout" name={name} />;
+  }
   */
```

Comment:
Added `layoutList` to this todo as well. Spreading `layoutList` or `primitiveList` above causes an error in the custom 404 page.

![image](https://user-images.githubusercontent.com/1350081/76921392-95be9d80-6893-11ea-8176-39908ff3e3de.png)

Reply:
This should be fixed now.